jax/random.py

```diff
@@ -82,10 +82,10 @@ def PRNGKey(seed: int) -> jnp.ndarray:
   if isinstance(seed, (int, np.ndarray)):
     # Special handling of raw integer values, which may have be 64bit even
     # when jax_enable_x64=False and we don't want to drop the top 32 bits
-    k1 = convert(np.bitwise_and(np.right_shift(seed, 32), 0xFFFFFFFF))
+    k1 = convert(np.bitwise_and(np.right_shift(seed, 32), np.uint32(0xFFFFFFFF)))
```

Comment:
Out of curiosity, what caused this change?

Reply:
See that commit.

![image](https://user-images.githubusercontent.com/1279292/99018216-86affc80-2594-11eb-868e-ecc108efe223.png)

Simply put, `0xFFFFFFFF` does not fit into 4bytes signed integer and is automatically promoted to 8bytes

Reply:
https://github.com/google/jax/pull/4843/commits/094542f0f103c72e2d261c7474841d5f6d7da9d5

Reply:
@jakevdp as an FYI

Reply:
And I was curious why 0x7FFFFFFF is not 28bytes python object, then I found https://stackoverflow.com/a/10365639/2091555

Reply:
I'm still not sure why this would differ between windows and linux/mac, however. That said, the change seems fine.